src/utils/init/config-manual.js

```diff
@@ -21,70 +16,27 @@ module.exports = async function configManual(ctx, site, repo) {
   ])
 
   if (!sshKeyAdded) {
-    ctx.exit()
+    exit()
   }
+}
 
-  repo.provider = 'manual'
-  repo.deploy_key_id = key.id
-
-  // TODO: Look these up and default to the lookup order
-  const { buildCmd, buildDir } = await inquirer.prompt([
-    {
-      type: 'input',
-      name: 'buildCmd',
-      message: 'Your build command (hugo build/yarn run build/etc):',
-      filter: (val) => (val === '' ? undefined : val),
-    },
+const getRepoPath = async ({ repoData }) => {
+  const { repoPath } = await inquirer.prompt([
     {
       type: 'input',
-      name: 'buildDir',
-      message: 'Directory to deploy (blank for current dir):',
-      default: '.',
+      name: 'repoPath',
+      message: 'The SSH URL of the remote git repo:',
+      default: repoData.url,
+      validate: (url) => SSH_URL_REGEXP.test(url) || 'The URL provided does not use the SSH protocol',
     },
   ])
 
-  const tomlpath = path.join(ctx.netlify.site.root, 'netlify.toml')
-  const tomlDoesNotExist = !fs.existsSync(tomlpath)
-  if (tomlDoesNotExist && (!ctx.netlify.config || Object.keys(ctx.netlify.config).length === 0)) {
-    const { makeNetlifyTOML } = await inquirer.prompt([
-      {
-        type: 'confirm',
-        name: 'makeNetlifyTOML',
-        message: 'No netlify.toml detected. Would you like to create one with these build settings?',
-        default: true,
-      },
-    ])
-    if (makeNetlifyTOML && ctx.netlify.site && ctx.netlify.site.root) {
-      fs.writeFileSync(tomlpath, makeNetlifyTOMLtemplate({ command: buildCmd, publish: buildDir }))
-    } else {
-      throw new Error('NetlifyCLIError: expected there to be a Netlify site root, please investigate', ctx.netlify.site)
-    }
-  }
-
-  repo.dir = buildDir
-
-  if (!repo.repo_path) {
```

Comment:
This was kind of duplicated with https://github.com/netlify/cli/blob/bd59e01b8f4214035e6f69bd85182008bb810f07/src/commands/sites/create.js#L121

with the exception of not letting the user chose a URL if it was parsed successfully.

I opted for always letting the user confirm the URL, and using the parsed value as a default